### PR TITLE
Example fixes

### PR DESCRIPTION
**Pull request by farhatcode**
I happened to come across `tabs.previous` intent in tabs.toml, that match could be improved. Then gave a quick glance to all other intents and come up with another example correction.

### CODE REVIEW

File: extension/intents/tabs/tabs.toml [hunk 1]
```diff
@@ -205,7 +205,7 @@ phrase = "Reset zoom"
 [tabs.previous]
 description = "Go to a previously active tab"
 match = """
-  (Go | switch | change | activate |) (the | my |) previous tab
+  (Go | switch | change | activate |) (to |) (the | my |) previous tab
```
Comment (ianb) [inline extension/intents/tabs/tabs.toml]: That would match "go to my previous tab" or "switch to my previous tab" etc. So I think we'd want to keep it.

Comment (farhatcode) [reply extension/intents/tabs/tabs.toml]: I have updated it @ianb.

Comment (farhatcode) [reply extension/intents/tabs/tabs.toml]: or may I close this PR because this change was not required?

**Comment by ianb** [comment]
@farhatcode Yes, it occurs to me that the testing example probably didn't need updating – I think it was something that I actually encountered as a mistranscription and wanted to be sure it worked. There's even some special code for repeated words: https://github.com/mozilla/firefox-voice/blob/master/extension/background/language/textMatching.js#L94-L100

**Comment by farhatcode** [comment]
Thanks @ian for explaining.

**Pull request closed without merging**